__tests__/app.test.js

```diff
@@ -303,4 +303,26 @@ describe("DELETE /api/comments/:comment_id", () => {
           expect(body.msg).toBe("Bad request: Invalid input");
         });
       });
-  });
\ No newline at end of file
+  });
+
+  //TASK 10
+  describe('GET /api/users', () => {
+    test('200: serves an array of all users', () => {
+      return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then(({body}) => {
+        const { users } = body;
+        expect(Array.isArray(users)).toBe(true);
+        users.forEach((user) => {
+          expect(user).toEqual(
+            expect.objectContaining({
+              username: expect.any(String),
+              name: expect.any(String),
+              avatar_url: expect.any(String),
+            })
+          );
```

Comment:
You should check the length of the array - otherwise an empty array would pass this test (the forEach would skip)